README.md

```diff
@@ -317,6 +317,45 @@ now keep this in mind:
 The CLI text editor we all know, just in case you need to inspect some bug in
 hot deployments.
 
+### `install-addons`
+
+This script will iterate the `addons.yml` file to gather support addons, and
+subsequently install to the Odoo instance. It will then exit.
+
+This script is great to use during buildout in order to prepare an instance.
+Below is a minimalistic `docker-compose` file illustrating a use case:
+
+```yaml
+version: '2'
+
+services:
+
+  install:
+    image: laslabs/odoo-buildouts:dev-10
+    command: 'install-addons'
+    tty: true
+    depends_on:
+      - db
+    tty: true
+
+  web:
+    image: laslabs/odoo-buildouts:dev-10
+    restart: unless-stopped
+    tty: true
+    depends_on:
+      - db
+      - install
+    ports:
+      - 8069
+      - 8071
+    environment:
+      WAIT_NOHOST: 'install'
+
+  db:
+    image: postgres:9.6-alpine
```

Comment:
Right, but the problem is that I will have to maintain the example, because if at some point the client libraries get upgraded, it could work no more.

I imagine an issue saying: "I followed this example and it does not work"; me looking at the example, it's wrong, update version...

That variable is self-descriptive enough to make the example reader what it does, so please use it.